src/uct/cuda/cuda_copy/cuda_copy_md.h

```diff
@@ -35,6 +35,7 @@ typedef struct uct_cuda_copy_md {
         int                      dmabuf_supported;
         ucs_ternary_auto_value_t enable_fabric;
         uct_cuda_pref_loc_t      pref_loc;
+        int                      cuda_async_managed;
```

Comment:
minor: wouldn't it be more consistent to have also `ucs_memory_type_t` here?

Reply:
not sure why it is better. We cache/store on md only what is needed. Also we do not need to check memory types again in memory detection flow

Reply:
i think it should be ucs_memory_type_t because this is what the parser expects for enums and also we used it for rndv_frag_mem_type

Reply:
this is not a config field (so parser and rndv_frag_mem_type are not relevant), i use `ucs_memory_type_t` for config. This is what we save on MD for later evalution. IMO, having  a boolean looks better.